### PR TITLE
resolved meeting_number template bug but some problems are there in m…

### PR DESCRIPTION
**Pull request by ayesha54**
…eeting invitation template

### CODE REVIEW

File: app/views/user_mailer/send_meeting_invitation.slim [hunk 1]
```diff
@@ -9,12 +9,13 @@ tr
                 tr
                   td.mcnTextContent style=("padding-top:0; padding-right:18px; padding-bottom:9px; padding-left:18px;") valign="top" 
                     div style=("text-align: left;") 
-                      span style="color:#808080"  Hello #{@invitation.invitee_email},
+                    debugger
```
Comment (mohsinali) [inline app/views/user_mailer/send_meeting_invitation.slim]: Why is this here?

**Comment by ayesha54** [comment]
Sir talha will let me know after testing this then I will solve this

File: app/views/user_mailer/send_meeting_invitation.slim (hunk 1 above)
Comment (ayesha54) [reply app/views/user_mailer/send_meeting_invitation.slim]: Ok sir I'll check and resolve this issue .
